src/app/src/components/Layers/ParcelLayer.js

```diff
@@ -0,0 +1,69 @@
+import { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { DynamicMapLayer } from 'esri-leaflet';
+
+import { useMapLayer } from '../../hooks';
+import { PARCELS_LAYER_URL, DATA_LAYERS } from '../../constants';
+
+const PARCELS_LAYER_STYLE = [
+    {
+        id: 0,
+        source: {
+            type: 'mapLayer',
+            mapLayerId: 0,
+        },
+        drawingInfo: {
+            renderer: {
+                type: 'simple',
+                symbol: {
+                    type: 'esriSMS',
+                    style: 'esriSMSDiamond',
+                    color: [183, 121, 31, 255], // var(--chakra-colors-yellow-600)
```

Comment:
It would be great to get this from Chakra UI somehow, like via `useTheme` or something, but I couldn't figure out a way to do that, so just added a comment for reference.